Cargo.toml

```diff
@@ -57,10 +58,14 @@ lazy_static = { version = "1.4.0", optional = true }
 # gpu dependencies
 ec-gpu = { version = "0.1.0", optional = true }
 
+# serde dependencies
+serde = { version = "1.0.144", features = ["derive"], optional = true }
```

Comment:
Does this need to pin `serde 1.0.144`? I would prefer that we only specify the minimum required `serde` version, for maximal compatibility with downstream users. For my prior comment, this would mean a version of `serde` that supports `{Serializer, Deserializer}::is_human_readable`.